oracle/cloud-config.yml

```diff
@@ -0,0 +1,64 @@
+azs:
+- name: z1
+  cpi: oracle-cpi-ref
+  cloud_properties:
+    availability_domain: ((ad1))
+- name: z2
+  cpi: oracle-cpi-ref
+  cloud_properties:
+    availability_domain: ((ad2))
+- name: z3
+  cpi: oracle-cpi-ref
+  cloud_properties:
+    availability_domain: ((ad3))
+
+vm_types:
+- name: default
+  cloud_properties:
+    instance_shape: VM.Standard1.2
+- name: vm4
```

Comment:
These cloud-configs are meant as a starting point for users. For that reason, we have only been including `default` and `large` as examples. This helps avoid users getting too tied to these provided `vm_types`, but still provides a reference example about how multiple `vm_types` can be used.

What do you think about dropping the `vm4` and `bm_default`, and adding a `large` one here?

Reply:
I agree keeping it simple is important, that's why not all available types are enumerated in there.  Renaming vm4 to large for consistency with other vendors makes sense; we'll do that. 

However, I'd like to keep the bm_default example to illustrate that users don't have to do anything extra for using baremetal (as opposed to virtual machines) instances in our cloud.  I remember couple of Pivotal platform architects asking that question at SpringOne last December.  

Can rename bm_default to baremetal if that helps.